ush/parsing_namelists_FV3.sh

```diff
@@ -0,0 +1,603 @@
+#! /bin/sh
+##### 
+## "parsing_namelist_FV3.sh"
+## This script writes namelist for FV3 model
+##
+## This is the child script of ex-global forecast,
+## writing namelist for FV3
+## This script is a direct execution.
+#####
+
+FV3_namelists(){
+
+# copy over the tables
+DIAG_TABLE_BASE=${DIAG_TABLE_BASE:-$PARM_FV3DIAG/diag_table}
+DIAG_TABLE=${DIAG_TABLE:-$PARM_FV3DIAG/diag_table_aod}
+DATA_TABLE=${DATA_TABLE:-$PARM_FV3DIAG/data_table}
+FIELD_TABLE=${FIELD_TABLE:-$PARM_FV3DIAG/field_table}
+
+# build the diag_table with the experiment name and date stamp
+if [ $DOIAU = "YES" ]; then
+cat > diag_table << EOF
+FV3 Forecast
+${gPDY:0:4} ${gPDY:4:2} ${gPDY:6:2} ${gcyc} 0 0
+EOF
+cat $DIAG_TABLE_BASE >> diag_table
+else
+cat > diag_table << EOF
+FV3 Forecast
+${sPDY:0:4} ${sPDY:4:2} ${sPDY:6:2} ${scyc} 0 0
+EOF
+cat $DIAG_TABLE_BASE >> diag_table
+fi
+
+# Append AOD diag table for MERRA2
+if [ $IAER = "1011" ]; then
+  cat $DIAG_TABLE >> diag_table
+fi
+
+$NCP $DATA_TABLE  data_table
+$NCP $FIELD_TABLE field_table
+
+cat > input.nml <<EOF
+&amip_interp_nml
+  interp_oi_sst = .true.
+  use_ncep_sst = .true.
+  use_ncep_ice = .false.
+  no_anom_sst = .false.
+  data_set = 'reynolds_oi'
+  date_out_of_range = 'climo'
+  $amip_interp_nml
+/
+
+&atmos_model_nml
+  blocksize = $blocksize
+  chksum_debug = $chksum_debug
+  dycore_only = $dycore_only
+  ccpp_suite = '$CCPP_SUITE'
+  fdiag = $FDIAG
+  fhmax = $FHMAX
+  fhout = $FHOUT
+  fhmaxhf = $FHMAX_HF     
+  fhouthf = $FHOUT_HF    
+  $atmos_model_nml
+/
+
+&diag_manager_nml
+  prepend_date = .false.
+  $diag_manager_nml
+/
+
+&fms_io_nml
+  checksum_required = .false.
+  max_files_r = 100
+  max_files_w = 100
+  $fms_io_nml
+/
+
+&mpp_io_nml
+  shuffle=${shuffle:-1}
+  deflate_level=${deflate_level:-1}
+/
+
+&fms_nml
+  clock_grain = 'ROUTINE'
+  domains_stack_size = ${domains_stack_size:-3000000}
+  print_memory_usage = ${print_memory_usage:-".false."}
+  $fms_nml
+/
+
+&fv_core_nml
+  layout = $layout_x,$layout_y
+  io_layout = $io_layout
+  npx = $npx
+  npy = $npy
+  ntiles = $ntiles
+  npz = $npz
+EOF
+
+if [ $cpl = .true. ]; then
+  cat >> input.nml << EOF
+  dz_min =  ${dz_min:-"6"}    ! no longer in develop branch
+  psm_bc = ${psm_bc:-"0"}    ! no longer in develop branch
```

Comment:
Two things here: 
1. Do we need the comment? 
2. ppm_bc defaults to 1 in f/c-c

Reply:
This is another change that was made to coupled just this week, though I can't find where I got the change from.

Reply:
Found it. Fanglin said to change it to 1 because of a change to dz_min = 2. So if the default is 6, I'm guessing this should stay 0.